release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -334,6 +334,28 @@ In the table, features are marked with the following statuses:
 [id="ocp-4-5-deprecated-features"]
 === Deprecated features
 
+[id="ocp-4-5-deprecation-of-operatorsources"]
+==== OperatorSources and CatalogSourceConfigs block cluster upgrades
+
+OperatorSources and CatalogSourceConfigs have been deprecated for several
+{product-title} releases. Starting in {product-title} 4.4, if there are any
+custom OperatorSources or CatalogSourceConfigs objects present on the cluster,
+the `marketplace` cluster Operator set an `Upgradeable=false` condition and
```

Comment:
I think you're mixing tenses here with "Starting..." and "are" vs. "set" and "issued". If this still occurs in 4.5, then I'd move everything to present tense.